templates/components/global-footer/global-footer.html.twig

```diff
@@ -4,6 +4,38 @@
  * Default theme implementation for the basic structure of the global stanford footer
  */
 #}
+
+<section class="contact-footer" style="background-image:url('https://www-tc.pbs.org/wnet/nature/files/2017/03/1007.jpg')";>
```

Comment:
This doesn't appear to be the same image as used in the mock ups. Can you change it to the appropriate image and include it in this theme. Please do not depend on third party external images for this graphic.